client/service/src/builder.rs

```diff
@@ -138,7 +138,7 @@ impl<R> RpcExtensionBuilder for NoopRpcExtensionBuilder<R> where
 {
 	type Output = R;
 
-	fn build(&self, _deny: sc_rpc::DenyUnsafe) -> Self::Output {
+	fn build(&self, _deny: sc_rpc::DenyUnsafe, _subscriptions: SubscriptionManager) -> Self::Output {
```

Comment:
It's not strictly required for all RPCs to share the same `SubscriptionManager` (just sharing the executor would do), but it's okay.

Reply:
Is there's any advantage/disadvantage to sharing?

Reply:
The id space is going to be shared between different subscriptions. We are using random ones, but still you can call:
```
let id = subscribeHeads();
unsubscribeJustifications(id);
```
and it will probably work fine, since the manager is the same.
I don't think it's a serious issue though.